mllib/src/test/scala/org/apache/spark/ml/recommendation/ALSSuite.scala

```diff
@@ -520,6 +488,81 @@ class ALSSuite
   }
 }
 
+class ALSCleanerSuite extends SparkFunSuite {
+  test("Clean shuffles") {
+    val conf = new SparkConf()
+    val localDir = Utils.createTempDir()
+    val tempDir = Utils.createTempDir()
+    def getAllFiles: Set[File] =
+      FileUtils.listFiles(localDir, TrueFileFilter.INSTANCE, TrueFileFilter.INSTANCE).asScala.toSet
+    try {
+      conf.set("spark.local.dir", localDir.getAbsolutePath)
+      conf.set("spark.shuffle.manager", "sort")
```

Comment:
I don't see any thing particular in this test that requires sort shuffle. I removed this line and the test still passed on my local.